common/src/main/resources/data/ad_astra/recipes/recipes/nasa_workbench.json

```diff
@@ -8,7 +8,7 @@
       "tag": "ad_astra_platform:steel_blocks"
     },
     "C": {
-      "item": "minecraft:crafting_table"
+      "tag": "forge:crafting_tables"
```

Comment:
needs a common tag

Reply:
forge:crafting_tables would be the common tag.
Unless you change it to: 
#ad_astra:crafting_tables
Then you could add a folder for forge tags having the file being in forge/tags/items/crafting_tables.json
Inside the json you would have #ad_astra:crafting_tables.

Thus supporting forge and fabric?